 2007-07-30  Roland McGrath  <roland@redhat.com>

	* manual/libc.texinfo: Update copyrights, formatting magic, and
	@dircategory.  From Karl Berry <karl@freefriends.org>.

===================================================================
RCS file: /cvs/glibc/libc/manual/libc.texinfo,v
retrieving revision 1.94
retrieving revision 1.95
diff -u -r1.94 -r1.95
--- libc/manual/libc.texinfo	2003/11/12 00:37:03	1.94
+++ libc/manual/libc.texinfo	2007/07/31 01:53:58	1.95
@@ -2,10 +2,10 @@
 @comment %**start of header (This is for running Texinfo on a region.)
 @setfilename libc.info
 @settitle The GNU C Library
-@setchapternewpage odd
+@c setchapternewpage odd
 
 @comment Tell install-info what to do.
-@dircategory GNU libraries
+@dircategory Software libraries
 @direntry
 * Libc: (libc).                 C library.
 @end direntry
@@ -18,39 +18,33 @@
 @comment %**end of header (This is for running Texinfo on a region.)
 
 @c sold 0.06/1.09, print run out 21may96
-@set EDITION 0.10
-@set VERSION 2.3.x
-@set UPDATED 2001-07-06
+@set EDITION 0.11
+@set VERSION 2.6
+@set UPDATED 2006-12-03
 @set ISBN 1-882114-55-8
 
-@ifnottex
+@copying
 This file documents the GNU C library.
 
 This is Edition @value{EDITION}, last updated @value{UPDATED},
-of @cite{The GNU C Library Reference Manual}, for Version @value{VERSION}.
+of @cite{The GNU C Library Reference Manual}, for version @value{VERSION}.
 
-Copyright (C) 1993, 1994, 1995, 1996, 1997, 1998, 1999, 2001, 2002, 2003 Free Software Foundation, Inc.
+Copyright (C) 1993, 1994, 1995, 1996, 1997, 1998, 1999, 2001, 2002,
+2003, 2007 Free Software Foundation, Inc.
 
 Permission is granted to copy, distribute and/or modify this document
-under the terms of the GNU Free Documentation License, Version 1.1 or
+under the terms of the GNU Free Documentation License, Version 1.2 or
 any later version published by the Free Software Foundation; with the
-Invariant Sections being "Free Software Needs Free Documentation" and
-"GNU Lesser General Public License", the Front-Cover texts being (a)
-(see below), and with the Back-Cover Texts being (b) (see below).  A
+Invariant Sections being ``Free Software Needs Free Documentation''
+and ``GNU Lesser General Public License'', the Front-Cover texts being
+``A GNU Manual'', and with the Back-Cover Texts as in (a) below.  A
 copy of the license is included in the section entitled "GNU Free
 Documentation License".
 
-
-(a) The FSF's Front-Cover Text is:
-
-     A GNU Manual
-
-(b) The FSF's Back-Cover Text is:
-
-     You have freedom to copy and modify this GNU Manual, like GNU
-     software.  Copies published by the Free Software Foundation raise
-     funds for GNU development.
-@end ifnottex
+(a) The FSF's Back-Cover Text is: ``You are free to copy and modify
+this GNU Manual. Buying copies from GNU Press supports the FSF in
+developing GNU and promoting software freedom.''
+@end copying
 
 @iftex
 @shorttitlepage The GNU C Library Reference Manual
@@ -71,37 +65,22 @@
 @center for version @value{VERSION}
 @page
 @vskip 0pt plus 1filll
-Copyright @copyright{} 1993, 1994, 1995, 1996, 1997, 1998, 2001, 2002 Free Software Foundation, Inc.
+@insertcopying
+
 @sp 2
 Published by the Free Software Foundation @*
 59 Temple Place -- Suite 330, @*
 Boston, MA 02111-1307 USA @*
-ISBN @value{ISBN} @*
-
-Permission is granted to copy, distribute and/or modify this document
-under the terms of the GNU Free Documentation License, Version 1.1 or
-any later version published by the Free Software Foundation; with the
-Invariant Sections being "Free Software Needs Free Documentation" and
-"GNU Lesser General Public License", the Front-Cover texts being (a)
-(see below), and with the Back-Cover Texts being (b) (see below).  A
-copy of the license is included in the section entitled "GNU Free
-Documentation License".
-
-(a) The FSF's Front-Cover Text is:
-
-     A GNU Manual
-
-(b) The FSF's Back-Cover Text is:
-
-     You have freedom to copy and modify this GNU Manual, like GNU
-     software.  Copies published by the Free Software Foundation raise
-     funds for GNU development.
+@c ISBN @value{ISBN} @*
 
+@sp 2
 Cover art for the Free Software Foundation's printed edition
 by Etienne Suvasa.
 
 @end titlepage
-@page
+
+@shortcontents
+@contents
 
 @ifnottex
 @node Top, Introduction, (dir), (dir)
@@ -150,6 +129,4 @@
 @printindex pg
 
 
-@shortcontents
-@contents
 @bye
